src/Auditor.jl

```diff
@@ -84,7 +84,7 @@ function audit(prefix::Prefix, src_name::AbstractString = "";
             readmeta(f) do oh
                 if !is_for_platform(oh, platform)
                     if verbose
-                        @warn("Skipping binary analysis of $(relpath(f, prefix.path)) (incorrect platform)")
+                        @warn("Skipping binary analysis of $(relpath(f, prefix.path)) (incorrect platform)", platform=triplet(platform))
```

Comment:
I don't like, this would make the auditor overly verbose in verbose mode (which is what I use most, and what's used in CI in Yggdrasil) where the platform is _already_ printed.

Reply:
Ok, I'll close it then. Personally I prefer it because it's easy to miss warnings in verbose mode, and even then you have to look for the platform in some other message.